examples/commands/pkglists.rst

```diff
@@ -173,4 +173,37 @@ We can compute a package list from this file, and then upload those artifacts to
     $ conan upload --list=pkglist.json -r=myremote -c
 
 
+Removing packages lists
+-----------------------
+
+It is also possible to first ``conan list`` and create a list of things to remove, and then remove them:
+
+.. code-block:: bash
+
+    # Removes everything from the cache
+    $ conan list *#* --format=json > pkglist.json
+    $ conan remove --list=pkglist.json  -c
+
+Note that in this case, the default patterns are different in ``list`` and ``remove``, because of the destructive nature of ``conan remove``:
+
+- When a recipe is passed to ``remove`` like ``conan remove zlib/1.2.13`` will remove the recipe of ``zlib/1.2.13`` and all of its binaries, because the binaries cannot live without the recipe.
```

Comment:
```suggestion
- When a recipe is passed to ``remove`` like ``conan remove zlib/1.2.13``, it will remove the recipe of ``zlib/1.2.13`` and all of its binaries, because the binaries cannot live without the recipe.
```